torch/optim/optimizer.py

```diff
@@ -100,6 +100,12 @@ def _dispatch_sqrt(x: float):  # float annotation is needed because of torchscri
     else:
         return math.sqrt(x)
 
+def _dispatch_abs(x: float):  # float annotation is needed because of torchscript type inference
+    if not torch.jit.is_scripting() and isinstance(x, torch.Tensor):
+        return x.abs()
+    else:
+        return abs(x)
```

Comment:
I improved dynamo's `abs` support in #110398 so this function shouldn't be needed any more.